### PR TITLE
improved header's readability

### PR DESCRIPTION
**Pull request by bmarianome**
Hey, trying to make my first open source contribution here. 

Added a few simple things to the header, and a link to the mobbex developer's portal in a description a few lines below.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,37 @@
-# Mobbex
-
-[![Version](https://img.shields.io/npm/v/mobbex.svg)](https://www.npmjs.org/package/mobbex)
-[![Build Status](https://travis-ci.org/mobbexco/nodejs.svg?branch=master)](https://travis-ci.org/github/mobbexco/nodejs)
-[![Downloads](https://img.shields.io/npm/dt/mobbex.svg)](http://npmjs.com/package/mobbex)
-[![License](https://img.shields.io/apm/l/vim-mode)](https://github.com/mobbexco/nodejs/blob/master/LICENSE)
+<p align='center'>
+  <a href='https://strapi.io/#gh-light-mode-only'>
```
Comment (ezegros) [inline README.md]: This link should be the Mobex webpage, not Strapi.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
+    <img src='https://framerusercontent.com/images/VdMB4xykL9RjMlCz8ngVkj2kDYM.svg' width='300' alt='Mobbex logo' />
```
Comment (ezegros) [inline README.md]: Maybe we can use the mobbex logo here. I think this src is not working.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,37 @@
...
+  <a href='https://mobbex.dev'>Ir a la documentación</a>
+</p>
+
+<p align='center'>
+  <a href='https://www.npmjs.org/package/mobbex'>
+    <img src='https://img.shields.io/npm/v/mobbex.svg' alt='NPM Version' />
+  </a>
+  <a href='https://travis-ci.org/github/mobbexco/nodejs'>
+    <img src='https://travis-ci.org/mobbexco/nodejs.svg?branch=master' alt='Build Status' />
+  </a>
```
Comment (ezegros) [inline README.md]: We can delete this. We are not using Travis and the unknown badge isn't nice

**Review by ezegros — COMMENTED** [review]
Thank you for your contribution! I left a few comments to review.

**Comment by bmarianome** [comment]
I just added the logo to the folder but 8kb doesn't seems good. I can reduce it's size and upload it to the cloud maybe?

**Comment by ezegros** [comment]
It must be an URL to the file rather than a relative path; otherwise, I think it won't work. The file can be uploaded to the repo, but it must be the URL in master. Another option is using the GitHub avatar: `https://avatars.githubusercontent.com/u/24828094?v=4` (you can edit size by adding &s=xxx). If we want to use another logo we can ask @romanmobbex or @jlmobbex for it.

**Comment by bmarianome** [comment]
Done.

Actually the last logo URL was working, it's not visible because it's white, like the browser's background. I took the url form the logo in the Mobbex website, on top left. Anyways, I changed it because it won't be visible in github's light mode.

<br />
Mobbex logo

**Review by ezegros — APPROVED** [review]
LGTM :shipit:

**Comment by sugawaydev** [comment]
Nice change! Thank you very much!

**Pull request merged**